Source/AtomicEditor/Application/AEEditorApp.cpp

```diff
@@ -158,9 +158,10 @@ void AEEditorApp::HandleJSError(StringHash eventType, VariantMap& eventData)
     String errFilename = eventData[P_ERRORFILENAME].GetString();
     //String errStack = eventData[P_ERRORSTACK].GetString();
     int errLineNumber = eventData[P_ERRORLINENUMBER].GetInt();
+    int errRealLineNumber = vm_->GetRealLineNumber("AtomicEditor/out/" + errFilename, errLineNumber);
```

Comment:
https://github.com/rsredsq/AtomicGameEngine/commit/906d84ad1c56991c7b18a3d7795eef6f12045c13 should also fix that too